src/components/overview/ImageGallery/ImageGallery-index.jsx

```diff
@@ -0,0 +1,15 @@
+import React, { useState, useEffect } from 'react';
+import DefaultView from './DefaultView.jsx';
+
+const ImageGallery = ({selectedStyle}) => {
```

Comment:
Not sure if I agree with the naming of this file... imo, the name of the file should always be the same as the component inside. But since we haven't actually decided on that, it's up to you.

Reply:
> package-lock.json should be deleted, as it is supposed to be on the gitignore. Otherwise, looks good!

=> I included package-lock.json in gitignore but it still shows up. Do I just delete package-lock.json file??

> 

I just changed the file name as the component name!

Reply:
> Do I just delete package-lock.json file??
That's what worked for me, yes.